sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -78,7 +90,15 @@ function createPropertiesFromSpan(span: ReadableSpan): [Properties, Measurements
   const measurements: Measurements = {};
 
   for (const key of Object.keys(span.attributes)) {
-    if (!(key.startsWith("http.") || key.startsWith("rpc.") || key.startsWith("db."))) {
+    if (
+      !(
+        key.startsWith("http.") ||
+        key.startsWith("rpc.") ||
+        key.startsWith("db.") ||
+        key.startsWith("peer.") ||
+        key.startsWith("net.")
+      )
+    ) {
       properties[key] = span.attributes[key] as string;
```

Comment:
Looking at [max lengths](https://docs.microsoft.com/en-us/azure/azure-monitor/app/data-model-request-telemetry#custom-properties), key and value should be truncated.

Reply:
I'm not adding truncation in this PR, will address that later, created a task for it